stable/database/templates/deployment.yaml

```diff
@@ -105,7 +105,7 @@ spec:
           subPath: {{ $key }}
         {{- end -}}
         {{- end }}
-        - name: logdir
+        - name: log-te-volume
           mountPath: /var/log/nuodb
```

Comment:
I don't see how collisions by multiple TEs in the Deployment are avoided.
Would 3 different TEs write to 3 different log files - or all share the same log file?

Reply:
The only thing that a TE ever writes to disk is a core file. We don't write any other log information. The log goes to `stdout` instead. Core files have a file format dependant on both PID (not so unique) and `hostname`. If the TE crashes multiple times on the same host, with the same PID, the core generator just appends a counter at the end.

Reply:
Ok - thanks!